sql/parser/eval.go

```diff
@@ -769,6 +769,8 @@ func (ctx *EvalContext) GetTxnTimestamp() roachpb.Timestamp {
 	return ctx.txnTimestamp
 }
 
+// SetTxnTimestamp associates the EvalContext to a specified
```

Comment:
No harm in using the verb `set` in the description (`associates` reads a bit odd to me).